packages/headless/config/search/index.ts

```diff
@@ -0,0 +1,5 @@
+const {ELASTICSEARCH_NODE, ELASTICSEARCH_INDEX } = process.env
+export const searchConfig = {
+    node: ELASTICSEARCH_NODE,
+    index: ELASTICSEARCH_INDEX
```

Comment:
Add example value for these. add these fields with example value in .env.example file

Reply:
Done